### DISCUSSION TITLE
Release 2.4.5 on PyPI

### DISCUSSION THREAD

**Discussion by dbartenstein** [farridav/django-jazzmin#279] [Q&A] [answered]
Hi there!

Is there any reason why [Jazzmin 2.4.5](https://github.com/farridav/django-jazzmin/releases/tag/v2.4.5) has not yet appeared on PyPI? PyPI still shows 2.4.4 as the most recent version: https://pypi.org/project/django-jazzmin/

**Comment by farridav** [comment]
Ah it's seems the you release failed due to the version number not being bumped properly https://github.com/farridav/django-jazzmin/runs/2240161468?check_suite_focus=true

I will sort this today

**Comment by farridav** [comment, marked as answer]
2.4.5 has now been released to pypi (https://pypi.org/project/django-jazzmin/2.4.5/)

I'm investigating some ways to improve the release process, currently the workflow is:

1. Bump the version in `pyproject.toml` and `jazzmin/__init__.py`
2. Push to master
3. Make a new release in github releases (With release notes)
4. Workflow is triggered that deploys the current version to pypi

Someday id like to find a way to do away with steps 1 & 2 (And still use semantic versioning)

**Reply by dbartenstein** [reply]
Thanks @farridav!

**Reply by PacificGilly** [reply]
This has been achieved now, so every new PR merge will release a new version to test PyPI (https://github.com/farridav/django-jazzmin/pull/550) 🎉
